tendrl/alerting/handlers/__init__.py

```diff
@@ -0,0 +1,224 @@
+import datetime
+import etcd
+import importlib
+import inspect
+import json
+import multiprocessing
+import os
+import six
+from tendrl.alerting import constants
+from tendrl.alerting.objects.alert import AlertUtils
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+import time
+
+
+class NoHandlerException(Exception):
+    pass
+
+
+class HandlerMount(type):
+
+    def __init__(cls, name, bases, attrs):
+        if not hasattr(cls, 'handlers'):
+            cls.handlers = []
+        else:
+            cls.register_handler(cls)
+
+    def register_handler(cls, handler):
+        instance = handler()
+        cls.handlers.append(instance)
+
+
+@six.add_metaclass(HandlerMount)
+class AlertHandler(object):
+
+    handles = ''
+    representive_name = ''
+
+    def __init__(self):
+        self.time_stamp = datetime.datetime.now().isoformat()
```

Comment:
all now() should be generated using https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/time_utils.py#L5